Watchman.Discord/WatchmanBot.cs

```diff
@@ -96,13 +98,24 @@ public WorkflowBuilder GetWorkflowBuilder()
                 });
         }
 
-        private void PrintDebugExceptionInfo(Exception e, Contexts contexts)
+        private void SendExceptionInfo(Exception e, Contexts contexts)
         {
             var exceptionMessage = this.BuildExceptionMessage(e).ToString();
             var messagesService = this._context.Resolve<MessagesServiceFactory>().Create(contexts);
+            messagesService.ChannelId = this._configuration.ExceptionChannelID;
+            var isBotException = e.InnerException is BotException;
+            if (isBotException && this._configuration.SendOnlyUnknownExceptionInfo)
+            {
+                return;
+            }
             messagesService.SendMessage(exceptionMessage, Devscord.DiscordFramework.Commons.MessageType.BlockFormatted);
         }
 
+        private void PrintDebugExceptionInfo(Exception e, Contexts contexts)
+        {
+            this.SendExceptionInfo(e, contexts);
```

Comment:
nie łącz tego, bo to są zupełnie dwie różne rzeczy
PrintDebugExceptionInfo nie zmieniaj